src/Components/Attempt/index.tsx

```diff
@@ -13,13 +15,20 @@ function Attempt(props: { isActive: boolean, action: Function, attemptColors: Ar
     //     (_, i) => <ColorContainer myColor={-props.attemptState[i]} selectedColor={props.selectedColor} action={props.action} />
     // )
 
+    const haveWhite = (color: string) => color === DEFAULT_PEG_COLOR;
+
     return (
         <div className="attemptContainer">
             <div className="attemptColorsContainer">
                 {colorComponents}
             </div>
+            <div className="checkSolutionContainer">
+                {
+                    (props.isActive && !props.attemptColors.some(haveWhite)) ? <CheckSolution checkSolutionAction={props.checkSolutionAction} /> : <></>
```

Comment:
instead of `<>/<>` we can simply use `null` here

Reply:
Done.